.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
It's good practice to avoid leaving sensitive information like tokens and API keys in the code. Consider using environment variables or a .env file to store these securely.